### DISCUSSION TITLE
Add a way to hide collections in some cases

### DISCUSSION THREAD

**Discussion by GiantappMan** [supabase/supabase#9615] [Feature Requests]
# Feature request

## Is your feature request related to a problem? Please describe.

According to this [document](https://supabase.com/docs/guides/integrations/directus), I integrated Directus and SupaBase. It works perfectly, but Directus creates many collections.
I don't want to expose these collections and I don't want to change the data in these collections from Supabase. So I want a way to hide these collections and not generate an API for these collections.

## Describe the solution you'd like

Add a way to hide these.

![image](https://user-images.githubusercontent.com/80653/192493014-f030392f-5eed-4713-8fb3-43e71bd94053.png)

**Comment by saltcod** [comment]
Hey @DaZiYuan Adding pagination here is in our backlog and we hope to get to it soon. Thanks for submitting this!
